packages/react-jss/src/utils/sheets-meta.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import type {StyleSheet} from 'jss'
+import type {StaticStyles} from '../types'
+
+interface SheetMeta {
+  styles: StaticStyles;
+  dynamicStyles: StaticStyles;
+  dynamicRuleCounter: number;
+}
+
+const sheetsMeta = new WeakMap<StyleSheet, SheetMeta>()
+
+function getMetaForSheet(sheet: StyleSheet) {
+  return sheetsMeta.get(sheet)
+}
+
+function addMetaForSheet(sheet: StyleSheet, meta: SheetMeta) {
```

Comment:
`setMeta` and `getMeta` makes more sense to me

Reply:
Done.